cred/src/main/java/org/opencadc/cred/ServiceAvailability.java

```diff
@@ -92,14 +91,6 @@ public boolean heartbeat() {
     public Availability getStatus() {
         boolean isGood = true;
         String note = "service is accepting requests";
-        try {
```

Comment:
The service requires a valid signing cert to function, so that should be tested here (using CheckCertificate).